educator/src/Dscp/DB/DSL/Interpret/SimpleTxDB.hs

```diff
@@ -142,4 +142,6 @@ runSimpleTxDBQuery dbTx dbObj query =
         cId5 = Course 5
 
 getTxCourseId :: PrivateTx -> Id Course
-getTxCourseId tx = _aCourseId (_sAssignment (_ssSubmission (_ptSignedSubmission tx)))
+getTxCourseId tx = _aCourseId (getAssignment (_ssSubmission (_ptSignedSubmission tx)))
+  where
+    getAssignment = error "Some magic should happen here here"
```

Comment:
But it doesn't happen. Is it OK?

Reply:
My feeling is that there is no need to maintain this module, but waiting for response of @Heimdell